apps/web/app/(app)/environments/[environmentId]/integrations/airtable/components/AddIntegrationModal.tsx

```diff
@@ -73,12 +75,24 @@ export const AddIntegrationModal = ({
   const [tables, setTables] = useState<TIntegrationAirtableTables["tables"]>([]);
   const [isLoading, setIsLoading] = useState(false);
   const { handleSubmit, control, watch, setValue, reset } = useForm<IntegrationModalInputs>();
+  const [includeHiddenFields, setIncludeHiddenFields] = useState(false);
+  const [includeMetadata, setIncludeMetadata] = useState(false);
```

Comment:
can we move these states in the react-hook-form implementation? in other integrations as well?

Reply:
Moved into a new ticket https://github.com/formbricks/internal/issues/233